packages/css/src/types.ts

```diff
@@ -642,7 +652,12 @@ export interface Theme {
   printColorModeName?: string
 
   /**
-   * Adds styles defined in theme.styles.root to the <body> element along with color and background-color
+   * Adds styles defined in theme.styles.root to the <html> element along with color and background-color
+   */
+  useRootStyles?: boolean
+
+  /**
+   * Legacy : Adds styles defined in theme.styles.root to the <body> element along with color and background-color
```

Comment:
```suggestion
   * @deprecated
   * Adds styles defined in theme.styles.root to the <body> element along with color and background-color
```

This should make `useBodyStyles` display with strikethrough in popular IDEs AFAIK (VSCode, and possible JetBrains).
There are also lint rules checking for usage of deprecated APIs which respect this JSDoc comment.